src/plugins/query_enhancements/public/search/sql_async_search_interceptor.ts

```diff
@@ -7,6 +7,8 @@ import { trimEnd } from 'lodash';
 import { BehaviorSubject, Observable, throwError } from 'rxjs';
 import { i18n } from '@osd/i18n';
 import { concatMap, map } from 'rxjs/operators';
+import { UiActionsStart } from 'src/plugins/ui_actions/public';
+import uuid from 'uuid';
```

Comment:
Let's use a specific version so if they change their default, we will still produce similar ones.
```suggestion
import { v4 as uuidv4 } from 'uuid';
```

Reply:
Updated -- though I'm not sure if it matters since they're short-lived uuids that are generated on a per-query-run basis and don't need to be persisted